examples/clipboard.rs

```diff
@@ -0,0 +1,92 @@
+use bevy::{input::common_conditions::input_just_pressed, prelude::*};
+use bevy_egui::{egui, EguiContexts, EguiPlugin};
+use egui::{text_edit::CursorRange, TextEdit, Widget};
+use wasm_bindgen_futures::spawn_local;
+use web_sys::ClipboardPermissionDescriptor;
+
+fn main() {
+    App::new()
+        .add_plugins(DefaultPlugins)
```

Comment:
You can do this to turn off prevent_defaults now if that helps:
```suggestion
        .add_plugins(DefaultPlugins.set(WindowPlugin {
            primary_window: Some(Window {
                prevent_default_event_handling: false,
                ..default()
            }),
            ..default()
        }))
```

Reply:
I think I want to do that, I'm still digesting what exactly is the behaviour "default" that is referred to, how it helps with copypaste and why doesn't it work out of the box with that.

Maybe it's just the managed clipboard getting in the way, not sure..

Reply:
It prevents event handling on the html canvas Bevy uses from bubbling up to the rest of the page, which prevents hotkeys like F5 to refresh from working.

If your implementation of the clipboard requires JS to be handling the hotkeys, I think it would need to be set to false so you would get the ctrl+c/v events. I haven't looked too much at what you have in this PR, it might not be needed.

Reply:
Thanks, indeed after a few tests, prevent_default_handling has to be `false`  in order to get the events :tada: ; my other issue was that I was registering to "paste" events on the canvas, but I had to do it on the document.

Reply:
If you got any other information about this issue, I look forward to your feedbacks :)